cmd/termdash/tasksource.go

```diff
@@ -27,7 +27,10 @@ type TaskSource struct {
 
 func NewTaskSource(tesHTTPServerAddress string, pageSize uint32) *TaskSource {
 	// init funnel http client
-	cli := client.NewClient(tesHTTPServerAddress)
+	cli, err := client.NewClient(tesHTTPServerAddress)
+	if err != nil {
+		panic(err)
```

Comment:
How hard is it to eliminate this panic? Seems like you're fixing one but adding another.

Reply:
I was able to eliminate all of the panics in this code in the latest commit.